jax/_src/xla_bridge.py

```diff
@@ -324,14 +325,57 @@ def discover_pjrt_plugins() -> None:
   method, which calls jax._src.xla_bridge.register_plugin with its plugin_name,
```

Comment:
Thanks Stella! Could you update the comment in `(1) be place in a root folder `jax_plugins` and follow other namespace package requirements` to include set up entry_point?

Reply:
Done